assets/js/components/ClusterDetails/AscsErsClusterDetails.jsx

```diff
@@ -0,0 +1,120 @@
+import React, { useState, useEffect } from 'react';
+
+import PageHeader from '@components/PageHeader';
+import BackButton from '@components/BackButton';
+
+import ListView from '@components/ListView';
+import ProviderLabel from '@components/ProviderLabel';
+import DottedPagination from '@components/DottedPagination';
+
+import ChecksComingSoon from '@static/checks-coming-soon.svg';
+
+import SBDDetails from './SBDDetails';
+import StoppedResources from './StoppedResources';
+
+function AscsErsClusterDetails({
+  clusterName,
+  cibLastWritten,
+  provider,
+  details,
+}) {
+  const [sapSystems, setSapSystems] = useState([]);
+  const [currentSapSystem, setSapSystem] = useState(null);
+
+  useEffect(() => {
+    setSapSystems(details?.sap_systems);
```

Comment:
I think you don't need a state variable for the `sapSystems`

Reply:
You are right!
Backported from other commit that will need (I think so XD). But it will come in a next PR.
Removing by now.